Appserver/Startup.cs

```diff
@@ -46,9 +46,12 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             app.UseHttpsRedirection();
             app.UseStaticFiles();
 
+            //app.UseCors(builder =>
+              //builder.WithOrigins("http://iddpwa.azurewebsites.net"));
+
             app.UseCors(builder =>
-              builder.WithOrigins("http://iddpwa.azurewebsites.net"));

```

Comment:
we should change this back to  `builder.WithOrigins("http://iddpwa.azurewebsites.net"))`